src/components/common/styles/AboutUs.scss

```diff
@@ -80,4 +77,185 @@
     .fa-github-square{
         color: black;
     }
+}
+
+// ========= Tablet View ===================
+@media (max-width:800px) {
+    .about-us-section {
+        display: block;
+        width: 95%;
+        margin-left: 10%;
+    }
+    
+    .card-container {
+        display: flex;
+        justify-content: center;
+        flex-direction: column;
+        .card{
+            margin-inline-start: 5%;
+            margin-block-end: 5%;
+        }
+    }
+    
+    .about-us-header {
+        background-image: url('./imgs/seattle-skyline.png');
+        background-repeat: no-repeat;
+        background-position: 100%;
+        background-size: cover;
+        height: 400px;
+        display: flex;
+        justify-content: space-around;
+        width: 100%;
+        h1 {
+            color: rgba(0, 0, 0, 0.7);
+            text-align: center;
+            font-weight: 700;
+            font-size: 4rem;
+            margin-left: 38%;
+            margin-block-start: 0.4em;
+            margin-block-end: 0.8em;
+        }
+    }
+    
+    
+    .member-pic {
+        width: 35vw;
+        img {
+            border-radius: 10px;
+            width: 100%;
+        }
+    }
+    
+    .member-info{
+        width: 85%;
+        p {
+            display: block;
+            margin-block-start: 0.2em;
+            margin-block-end: 0.7em;
+            width: 85%;
+            font-size: 1.2rem;
+        }
+        h3 {
+            display: block;
+            font-size: 2.5em;
+            margin-block-start: 0.5em;
+            margin-block-end: 0em;
+            font-weight: bold;
+        }
+    }
+    
+    .about-us-team-div {
+        margin-top: 3%;
+        margin-block-end: 1em;
+        h2{
+            font-size: 3.5em;
+            font-weight: bolder;
+            margin-bottom: 0;
+            margin-inline-start: 5%;
+        }
+        span{
+            font-size: 1.85em;
+            margin-inline-start: 5%;
+            color: black;
+        }
+    }
+    
+    .member-social {
+        cursor: pointer;
+        margin-block-end: 1em;
+        .fa-linkedin{
+            color: #0E76A8;
+            margin-left: 1.5%;
+        }
+        .fa-github-square{
+            color: black;
+        }
+    }
+}
+
+
+// ========= Mobile View ===================
```

Comment:
Mobile view was needed in the site, thanks for adding the about us mobile and tablet views!